checkST2110.js

```diff
@@ -51,15 +51,15 @@ a=recvonly
 a=group:DUP primary secondary
 m=video 50000 RTP/AVP 112
 c=IN IP4 239.100.9.10/32
-a=source-filter: incl IN IP4 239.100.9.10 192.168.100.2
+a=source-filter:incl IN IP4 239.100.9.10 192.168.100.2
```

Comment:
I raised this in the ongoing review of ST 2110-10. I don't know whether it will be corrected or not.

Reply:
Yeah I think it will be, but given this specific file is intended to catch obvious copies of the file from the standard I probably shouldn't have 'corrected' it in the first place.